data/sql/updates/pending_db_world/db-fix-npc_trainer

```diff
@@ -0,0 +1 @@
+DELETE FROM `npc_trainer` WHERE `ID` = '19063' AND `SpellID` = '-201026';
```

Comment:
Please name the file with a `.sql` extension. ❤️

Reply:
Sorry, I am still a green hand. I already rename it.